backend.go

```diff
@@ -67,6 +84,9 @@ func newBackend(config *bouncerConfig) (*backendCTX, error) {
 			return nil, fmt.Errorf("unexpected type '%T' for iptables context", tmpCtx)
 		}
 	case "nftables":
+		if runtime.GOOS != "linux" {
```

Comment:
error message should be about nftables not to confuse user :)

Reply:
Good catch 👍